src/Common/src/System/Collections/Generic/LargeArrayBuilder.cs

```diff
@@ -50,12 +50,21 @@ public LargeArrayBuilder(bool initialize)
         /// Use <see cref="Add"/> if adding to the builder is a bottleneck for your use case.
         /// Otherwise, use <see cref="SlowAdd"/>.
         /// </remarks>
+        public void Add(T item) => Add(item, limit: int.MaxValue);
```

Comment:
Since the old `Add(T)` was marked with `AggressiveInlining`, should this new version be marked too?